src/Core/ConfigurationParser.re

```diff
@@ -32,7 +32,7 @@ let parseStringList = json => {
     List.fold_left(
       (accum, item) =>
         switch (item) {
-        | `String(v) => [v, ...accum]
+        | `String(v) => List.rev_append(accum, [v])
```

Comment:
This reverses `accum` every time you append to it, so while it'll work for two items, with more items it will alternately append to the beginning and end. It is also very inefficient.

I think this could be solved by just using `List.fold_right` instead of `List.fold_left`, but even just reversing it after the fold would be more efficient. But this can also just use `List.filter_map`, which would be much simpler.

Reply:
Something like 

```reason
List.filter_map(fun | `String(v) => Some(v) | _ => None, items);
```

Reply:
thanks. resolved.